modules/igv/1.0/config/default.yaml

```diff
@@ -0,0 +1,118 @@
+lcr-modules:
+    
+    igv:
+
+        inputs:
+            # Available wildcards: {unix_group} {seq_type} {tumour_sample_id} {normal_sample_id} {pair_status} {genome_build}
```

Comment:
Is it okay to have `{unix_group}` as a wildcard?

Reply:
It is not a standard column in the lcr-modules schema and is something specific to gambl. There are ways to deal with this - can show some examples and we can talk about this more before lab meeting today 😄